[BUG] SPFx : Module not found: Error: Can't resolve '@microsoft/microsoft-graph-client/lib/es/middleware/MiddlewareUtil'

**Describe the bug**
When building a SPFx WebPart with Graph Toolkit, I got the following error:

```
[01:36:49] Starting subtask 'webpack'...
[01:36:54] Error - [webpack] 'dist':
./node_modules/@microsoft/mgt-proxy-provider/dist/es6/CustomHeaderMiddleware.js
Module not found: Error: Can't resolve '@microsoft/microsoft-graph-client/lib/es/middleware/MiddlewareUtil' in 'E:\Tests\react-graph\node_modules\@microsoft\mgt-proxy-provider\dist\es6'
resolve '@microsoft/microsoft-graph-client/lib/es/middleware/MiddlewareUtil' in 'E:\Tests\react-graph\node_modules\@microsoft\mgt-proxy-provider\dist\es6'
  Parsed request is a module
  using description file: E:\Tests\react-graph\node_modules\@microsoft\mgt-proxy-provider\package.json (relative path: ./dist/es6)
    Field 'browser' doesn't contain a valid alias configuration
```

**To Reproduce**
Steps to reproduce the behavior:
1. Init SPFx project with a WebPart
2. Execute `npm i @microsoft/mgt @microsoft/mgt-react @microsoft/rush-stack-compiler-3.7`
3. Update the `tsconfig.json` file to replace the `extends` parameter by the one mentioned [here](https://docs.microsoft.com/en-us/graph/toolkit/get-started/build-a-sharepoint-web-part#update-typescript-in-your-project)
4. Update the `HelloWorldWebPart.ts` page to setup the provider as described [here](https://docs.microsoft.com/fr-fr/graph/toolkit/get-started/build-a-sharepoint-web-part#add-the-sharepoint-provider)
4. Execute `gulp serve` (or `gulp bundle`)
5. See error

**Expected behavior**
The webpack compilation is supposed to work but raises an error.

**Environment (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Framework: SPFx v11.0
 - Context: Developing a SharePoint WebPart with SPFx
 - Version: try to run with versions 2.0.0 and 2.1.0-preview.44812ca

**Additional context**
I've seen that this bug was already reported (#825), but it's not working as expected in 2.0 version and beyond.

--- a/packages/mgt-components/package.json
+++ b/packages/mgt-components/package.json
@@ -37,9 +37,10 @@
     "setLicense": "gulp setLicense --cwd ."
   },
   "dependencies": {
+    "@microsoft/mgt-element": "*",
+    "@microsoft/microsoft-graph-client": "^2.2.1",
     "@microsoft/microsoft-graph-types": "^1.27.0",
     "@microsoft/microsoft-graph-types-beta": "github:microsoftgraph/msgraph-typescript-typings#beta",
-    "@microsoft/mgt-element": "*",
     "office-ui-fabric-core": "11.0.0"
   },
   "publishConfig": {

--- a/packages/mgt-element/package.json
+++ b/packages/mgt-element/package.json
@@ -32,7 +32,7 @@
     "postpack": "cpx *.tgz ../../artifacts"
   },
   "dependencies": {
-    "@microsoft/microsoft-graph-client": "^2.1.1",
+    "@microsoft/microsoft-graph-client": "^2.2.1",
     "lit-element": "^2.4.0",
     "idb": "^5.0.7"
   },

--- a/packages/providers/mgt-msal-provider/package.json
+++ b/packages/providers/mgt-msal-provider/package.json
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@microsoft/mgt-element": "*",
+    "@microsoft/microsoft-graph-client": "^2.2.1",
     "msal": "^1.4.4"
   },
   "publishConfig": {

--- a/packages/providers/mgt-proxy-provider/package.json
+++ b/packages/providers/mgt-proxy-provider/package.json
@@ -38,7 +38,8 @@
     "postpack": "cpx *.tgz ../../../artifacts"
   },
   "dependencies": {
-    "@microsoft/mgt-element": "*"
+    "@microsoft/mgt-element": "*",
+    "@microsoft/microsoft-graph-client": "^2.2.1"
   },
   "publishConfig": {
     "directory": "dist"

--- a/packages/providers/mgt-teams-provider/package.json
+++ b/packages/providers/mgt-teams-provider/package.json
@@ -42,7 +42,8 @@
   },
   "dependencies": {
     "@microsoft/mgt-element": "*",
-    "@microsoft/mgt-msal-provider": "*"
+    "@microsoft/mgt-msal-provider": "*",
+    "@microsoft/microsoft-graph-client": "^2.2.1"
   },
   "publishConfig": {
     "directory": "dist"

